src/app/components/lobby/dialogs/open-game-popup/open-game-popup.component.ts

```diff
@@ -116,37 +126,39 @@ export class OpenGamePopupComponent implements OnInit {
 
   private loadTileSetData(id: number) {
     const req = this.httpClient.get<APIResponse<TileSet>>(OpenGamePopupComponent.requestUrl + `:id?id=${id}`);
-    req.subscribe((res: APIResponse<TileSet>) => {
-      const ts = res.payload;
-      this.selectedDeckExt = ts;
-      console.log('set data', this.selectedDeckExt);
-      this.generateTileListAsSet();
-    }, (error: any) => {
-      this.selectedDeckExt = undefined;
-      console.error('couldnt retrieve list of available decks', error);
-    });
+    req.subscribe(
+      (res: APIResponse<TileSet>) => {
+        const ts = res.payload;
+        this.selectedDeckExt = ts;
+        console.log('set data', this.selectedDeckExt);
+        this.generateTileListAsSet();
+      },
+      (error: any) => {
+        this.selectedDeckExt = undefined;
+        console.error('couldnt retrieve list of available decks', error);
+      }
+    );
   }
 
-  public closeMe() {
+  public closeMe(): void {
     this.dialogRef.close();
   }
 
   public createGame() {
     if (this.roomName !== undefined) {
       this.dialogRef.close({
         roomName: this.roomName,
-        skinName: (this.skinName !== undefined && this.skinName.length > 0) ? this.skinName : 'default',
-        randomizeTiles: this.randomizeTiles || false
+        tileSetId: this.deckList[this.selectedDeck].id || 1,
```

Comment:
dont really understand why this resolution is nessecary? I thought this.selectedDeck was already the Id, why do we need to resolve it again?